app/Http/Controllers/SuggestionController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Suggestion;
+use Illuminate\Http\Request;
+use Illuminate\Http\JsonResponse;
+
+class SuggestionController extends Controller
+{
+
+    public function index(): JsonResponse
+    {
+        return response()->json(Suggestion::all());
+    }
+
+    public function store(Request $request): JsonResponse
+    {
+        $suggestion = new Suggestion();
+        $suggestion->name = $request->name;
+        $suggestion->description = $request->description;
+        $suggestion->status = 'pending';
+        $suggestion->user_id = auth()->user()->id;
+        $suggestion->kebab_place_id = $request->kebabPlace;
+        $suggestion->save();
+        return response()->json("Dodano sugestie", 201);
+    }
+
+    public function updateState(Request $request, Suggestion $suggestion): JsonResponse
+    {
+        $suggestion->status = $request->status;
+        $suggestion->save();
+        return response()->json("Zaktualizowano status");
+    }
+
+    public function softDelete(Suggestion $suggestion): JsonResponse
```

Comment:
If you want to soft delete this record then you should add trait `use softDeletes `in model and `$table->softDeletes();` in migration

More about soft deletes: https://laravel.com/docs/11.x/eloquent#soft-deleting